tests/baselines/reference/neverIntersectionNotCallable.types

```diff
@@ -12,8 +12,8 @@ declare const f: { (x: string): number, a: "" } & { a: number }
 >  : ^^^^^^
 
 f()
->f() : any
->    : ^^^
+>f() : number

```

Comment:
The function itself is still not callable here but that erroneous call shows a better return type now. This is caused by `getSingleSignature` handling structured types now (an intersection in this very case)